src/main/java/com/gpperhour/GPPerHourPlugin.java

```diff
@@ -215,6 +216,7 @@ public class GPPerHourPlugin extends Plugin
 	private Widget inventoryWidget;
 	private ItemContainer inventoryItemContainer;
 	private ItemContainer equipmentItemContainer;
+	private ItemContainer rewardsItemContainer;
```

Comment:
probably no point in adding this as a field if we only reference it right after we assign it